mozdownload/scraper.py

```diff
@@ -244,6 +244,62 @@ def detect_platform(self):
         else:
             return "%s%d" % (mozinfo.os, mozinfo.bits)
 
+    @property
+    def checksums_binary(self):
```

Comment:
I would suggest that we introduce a `filename` property to the suggested `checksum` class which would return the filename.